debugger/loadprinter.ml

```diff
@@ -101,15 +101,15 @@ let match_printer_type desc typename =
   let printer_type =
     match
       Env.find_type_by_name
-        (Ldot(Lident "Topdirs", typename)) Env.empty
+        (Ldot(Lident "Topdirs", typename)) Env.initial_safe_string
```

Comment:
I think of this one as basically just being a bug fix (if one that couldn't really be hit before).  If I remember correctly, the typechecking for installing debug printers was sometimes failing because the environment was missing layout information for pervasives, and this fixed it.